metadata-ingestion/src/datahub/ingestion/source/kafka_connect.py

```diff
@@ -867,33 +884,48 @@ def get_dataset_table_for_topic(
             table = self.sanitize_table_name(table)
         return f"{dataset}.{table}"
 
+    def apply_transformations(
+        self, topic: str, transforms: List[Dict[str, str]]
+    ) -> str:
+        from java.util.regex import Pattern
```

Comment:
Why are you unable to use python `re` package?

Reply:
I have opted for this approach simply because it's consistent with how ConfluentJDBCSourceConnector and other connectors and sinks apply transformations.

We can use the re as well

Reply:
Changed this logic to use re.